ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -72,16 +72,22 @@ StatusDialog {
         popup.store.authenticateAndTransfer(d.uuid)
     }
 
-    property var recalculateRoutesAndFees: Backpressure.debounce(popup, 600, function() {
-        if(!!popup.selectedAccount && !!popup.selectedAccount.address && !!holdingSelector.selectedItem
-                && recipientInputLoader.ready && (amountToSend.ready || d.isCollectiblesTransfer)) {
-            popup.isLoading = true
-            d.routerError = ""
-            d.routerErrorDetails = ""
+    property var debounceRecalculateRoutesAndFees: Backpressure.debounce(popup, 1000, function() {
```

Comment:
same here, this could be an internal function?

Reply:
makes sense!

Reply:
done